lago/prefix.py

```diff
@@ -1225,6 +1226,7 @@ def _deploy_host(self, host):
                 return
 
             with LogTask('Wait for ssh connectivity'):
+                time.sleep(5)
```

Comment:
@david-caro - see latest revision. I've removed the wait - moved it to ovirt-system-tests - after it starts the VM, ./run_suite.sh simply sleep for 5 seconds. Brings us the same effect, without touching the deploy part in Lago.